lib/worker-pipeline.js

```diff
@@ -81,19 +88,24 @@ class AssetComputeWorkerPipeline {
             output.type = mimetype && mimetype.toLowerCase();
         }
 
-        // source.mimetype will always exist, no need to cultivate here.
-        const type = input.mimetype && input.mimetype.toLowerCase();
-        // make source mimetypes align to Manifest
-        const SOURCE_TYPE_MAP = {
-            "image/jpg" : "image/jpeg",
-            "image/tif" : "image/tiff"
-        };
+        // if source.mimetype does not exist, or it does not match the extension,
+        // try to find mimetype by mapping the extension
+        // this can happen if the client (for example the devtool) does not define the content-type correctly
+        let type;
+        if ((!input.mimetype || input.mimetype === DEFAULT_MIMETYPE) && input.name) {
+            const inputExtension = path.extname(input.name);
```

Comment:
Please add a log that we made the lookup (in case we need to debug something related to synonyms in extensions or mimetypes, so we know if it came from us or the client).